fem/restriction.hpp

```diff
@@ -57,6 +57,14 @@ class ElementRestriction : public Operator
        emulate SetSubVector and its transpose on GPUs. This method is running on
        the host, since the `processed` array requires a large shared memory. */
    void BooleanMask(Vector& y) const;
+
+   /// Fill a Sparse Matrix with Element Matrices.
+   void FillSpMat(const Vector &mat_ea, SparseMatrix &mat) const;
+
+   /// Fill the I array of a Sparse Matrix
```

Comment:
This comment (and one below it) are not very useful IMO. Maybe say something like `Fill I array of SparseMatrix corresponding to assembled matrix on L-vectors for this ElementRestriction.`

Reply:
What do you think of `Fill the I array of SparseMatrix corresponding to the sparsity pattern given by this ElementRestriction`?

Reply:
That's fine.

Reply:
Updated, tell me if you like it.